doc/third-party/fivetran/setup-guide.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Setup Guide
+title: Materialize Setup Guide
+description: Follow the guide to set up Materialize as a destination.
+---
+
+# Materialize Setup Guide {% badge text="Partner-Built" /%} {% badge text="Private Preview" /%}
+
+Follow the setup guide to connect Materialize to Fivetran.
+
+> NOTE: This destination is [partner-built](/docs/partner-built-program). For any questions related to Materialize destination and its documentation, contact [Materialize Support](mailto:support@materialize.com).
+
+---
+
+## Prerequisites
+
+To connect Materialize to Fivetran, you need the following:
+
+- A Materialize account with `CREATE` privileges on the database you're connecting to.
```

Comment:
```suggestion
- A Materialize account. If you already have one — great! If not, [sign up for a playground account](https://materialize.com/register/) first.
```

Reply:
It feels like the wrong place to mention the `CREATE` privilege requirement!

Reply:
Makes sense, updated!